ui-react/src/components/topBar/ReportProblemModal.tsx

```diff
@@ -0,0 +1,248 @@
+import React, { useState } from "react";
+import { Button, Form, Modal, Row, Col, Stack } from "react-bootstrap";
+import { openExternal } from "../../eventsBridge";
+import { repository } from "../../../package.json";
+import os from "os";
+import LogsCanvas from "./LogsCanvas";
+
+type ReportProblemForm = {
+  title?: string;
+  email?: string;
+  details?: string;
+};
+
+type ReportProblemModalProps = {
+  show: boolean;
+  handleCloseModal: () => {};
+};
+
+function ReportProblemModal({
+  show,
+  handleCloseModal,
```

Comment:
You can call it just `onClose`, you don't need to add the `modal` since it will be used always with the `modal` context anyway.